src/data/items.js

```diff
@@ -117,6 +118,17 @@ export const scarecrow = freeze({
   value: 160,
 })
 
+// TODO: move this, name it something better.
+export const mined = freeze({
+  description: 'a plot that has been mined. will be restored tomorrow',
+  hoveredPlotRangeSize: 1,
+  id: 'mined',
+  isReplantable: false,
+  name: 'Rubble',
+  type: MINED,
+  value: 0,
+})
```

Comment:
Your TODO hints to as much, but just to add in case it's helpful: The shoveling feature shouldn't be conceptually concerned with any items, just plot states.

Let me know if that's too vague to be helpful! 😅